lib/core/src/lib/services/people-process.service.spec.ts

```diff
@@ -65,7 +67,7 @@ describe('PeopleProcessService', () => {
             jasmine.Ajax.uninstall();
         });
 
-        it('should be able to retrieve people to involve in the task', (done) => {
+        it('should be able to retrieve people to involve in the task', async () => {
```

Comment:
it's hard to follow the pattern, you change "done" to async in one cases, and then change to "fakeAsync" in other

Reply:
why it's async/await here, and next tests with similar cases are "fakeAsync" ?

Reply:
it was done by mistake, fixed